### PR TITLE
Added Kafka Bridge tracing documentation

### PR DESCRIPTION
**Pull request by ppatierno**
### Type of change

- Documentation

### Description

This PR adds the documentation about tracing support for the Kafka Bridge added in #2122 

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Update/write design documentation in `./design`
- [ ] Write tests
- [ ] Make sure all tests pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: documentation/assemblies/assembly-distributed-tracing.adoc [hunk 1]
```diff
@@ -29,7 +29,7 @@ To set up distributed tracing for {ProductName}, follow these procedures:
 
 * xref:proc-instrumenting-kafka-streams-with-tracers-{context}[Instrument Kafka Streams applications for tracing]
 
-* xref:assembly-setting-up-tracing-mirror-maker-connect-{context}[Set up tracing for Mirror Maker and Kafka Connect]
+* xref:assembly-setting-up-tracing-mirror-maker-connect-bridge-{context}[Set up tracing for Mirror Maker, Kafka Connect and Kafka Bridge]
```
Comment (scholzj) [inline documentation/assemblies/assembly-distributed-tracing.adoc]: I think thi sis the usual comments fomr Paul / Daniel ;-)

```suggestion
* xref:assembly-setting-up-tracing-mirror-maker-connect-bridge-{context}[Set up tracing for Mirror Maker, Kafka Connect, and Kafka Bridge]
```

File: documentation/assemblies/assembly-setting-up-tracing-mirror-maker-connect-bridge.adoc [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
+Distributed tracing is supported for Mirror Maker, Kafka Connect (including Kafka Connect with Source2Image support) and Kafka Bridge.
 
 For Mirror Maker, messages are traced from the source cluster to the target cluster; the trace data records messages entering and leaving the Mirror Maker component.
 
 Only messages produced and consumed by Kafka Connect itself are traced. To trace messages sent between Kafka Connect and external systems, you must configure tracing in the connectors for those systems. For more information, see xref:assembly-deployment-configuration-kafka-connect-{context}[].
 
-include::../modules/proc-enabling-tracing-in-connect-mirror-maker-resources.adoc[leveloffset=+1]
\ No newline at end of file
+For Kafka Bridge, messages produced and consumed are traced as well as the HTTP requests, coming from clients, about sending and receiving messages through the bridge itself.
+In order to have and end to end tracing, you must configure tracing in your HTTP clients.
```
Comment (PaulRMellor) [inline documentation/assemblies/assembly-setting-up-tracing-mirror-maker-connect-bridge.adoc]:
```suggestion
In order to have and end-to-end tracing, you must configure tracing in your HTTP clients.
```

File: documentation/assemblies/assembly-setting-up-tracing-mirror-maker-connect-bridge.adoc [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
-= Setting up tracing for Mirror Maker and Kafka Connect
+[id='assembly-setting-up-tracing-mirror-maker-connect-bridge-{context}']
+= Setting up tracing for Mirror Maker, Kafka Connect and Kafka Bridge
 
-Distributed tracing is supported for Mirror Maker and Kafka Connect (including Kafka Connect with Source2Image support). 
+Distributed tracing is supported for Mirror Maker, Kafka Connect (including Kafka Connect with Source2Image support) and Kafka Bridge.
 
 For Mirror Maker, messages are traced from the source cluster to the target cluster; the trace data records messages entering and leaving the Mirror Maker component.
 
 Only messages produced and consumed by Kafka Connect itself are traced. To trace messages sent between Kafka Connect and external systems, you must configure tracing in the connectors for those systems. For more information, see xref:assembly-deployment-configuration-kafka-connect-{context}[].
```
Comment (laidan6000) [inline documentation/assemblies/assembly-setting-up-tracing-mirror-maker-connect-bridge.adoc]: Add a third sub-heading:

.Tracing in the the Kafka Bridge

File: documentation/assemblies/assembly-setting-up-tracing-mirror-maker-connect-bridge.adoc (hunk 2 above)
Comment (laidan6000) [inline documentation/assemblies/assembly-setting-up-tracing-mirror-maker-connect-bridge.adoc]:
```suggestion
In order to achieve end-to-end tracing of messages, you need to configure distributed tracing in your HTTP clients.
```

File: documentation/modules/proc-enabling-tracing-in-connect-mirror-maker-bridge-resources.adoc [hunk 4]
```diff
@@ -1,19 +1,21 @@
 // Module included in the following assemblies:
 //
-// assembly-setting-up-tracing-mirror-maker-connect.adoc
+// assembly-setting-up-tracing-mirror-maker-connect-bridge.adoc
 
-[id='proc-enabling-tracing-in-connect-mirror-maker-resources-{context}']
-= Enabling tracing in Mirror Maker and Kafka Connect resources
+[id='proc-enabling-tracing-in-connect-mirror-maker-bridge-resources-{context}']
+= Enabling tracing in Mirror Maker, Kafka Connect and Kafka Bridge resources
```
Comment (laidan6000) [inline documentation/modules/proc-enabling-tracing-in-connect-mirror-maker-bridge-resources.adoc]:
```suggestion
= Enabling tracing in Mirror Maker, Kafka Connect, and Kafka Bridge resources
```

File: documentation/modules/proc-enabling-tracing-in-connect-mirror-maker-bridge-resources.adoc [hunk 5]
```diff
@@ -1,19 +1,21 @@
...
 
-Update the configuration of `KafkaMirrorMaker`, `KafkaConnect`, and `KafkaConnectS2I` custom resources to specify and configure a Jaeger tracer service for each resource. Updating a tracing-enabled resource in your Kubernetes cluster triggers two events:
+Update the configuration of `KafkaMirrorMaker`, `KafkaConnect`, `KafkaConnectS2I`, and `KafkaBridge` custom resources to specify and configure a Jaeger tracer service for each resource. Updating a tracing-enabled resource in your Kubernetes cluster triggers two events:
 
-* Interceptor classes are updated in the integrated consumers and producers in Mirror Maker or Kafka Connect.
+* Interceptor classes are updated in the integrated consumers and producers in Mirror Maker, Kafka Connect or Kafka Bridge.
```
Comment (laidan6000) [inline documentation/modules/proc-enabling-tracing-in-connect-mirror-maker-bridge-resources.adoc]:
```suggestion
* Interceptor classes are updated in the integrated consumers and producers in Mirror Maker, Kafka Connect, or the {ProductName} Kafka Bridge.
```

File: documentation/modules/proc-enabling-tracing-in-connect-mirror-maker-bridge-resources.adoc [hunk 6]
```diff
@@ -1,19 +1,21 @@
...
 
-* The tracing agent initializes a Jaeger tracer based on the tracing configuration defined in the resource.
+* For Mirror Maker and Kafka Connect, the tracing agent initializes a Jaeger tracer based on the tracing configuration defined in the resource.
+
+* For Kafka Bridge, a Jaeger tracer based on the tracing configuration defined in the resource is initialized by the bridge itself.
```
Comment (laidan6000) [inline documentation/modules/proc-enabling-tracing-in-connect-mirror-maker-bridge-resources.adoc]:
```suggestion
* For the Kafka Bridge, a Jaeger tracer based on the tracing configuration defined in the resource is initialized by the Kafka Bridge itself.
```

File: documentation/modules/proc-enabling-tracing-in-connect-mirror-maker-bridge-resources.adoc [hunk 7]
```diff
@@ -62,6 +64,29 @@ spec:
   tracing:
     type: jaeger
 #...
+
+----
+.Jaeger tracer configuration for Kafka Bridge
```
Comment (laidan6000) [inline documentation/modules/proc-enabling-tracing-in-connect-mirror-maker-bridge-resources.adoc]:
```suggestion
.Jaeger tracer configuration for the Kafka Bridge
```

File: documentation/modules/proc-enabling-tracing-in-connect-mirror-maker-bridge-resources.adoc [hunk 8]
```diff
@@ -62,6 +64,29 @@ spec:
...
+[source,yaml,subs=attributes+]
+----
+apiVersion: kafka.strimzi.io/v1alpha1
```
Comment (laidan6000) [inline documentation/modules/proc-enabling-tracing-in-connect-mirror-maker-bridge-resources.adoc]: Should the API version be `v1beta1`?

File: documentation/assemblies/assembly-setting-up-tracing-mirror-maker-connect-bridge.adoc [hunk 9]
```diff
@@ -1,13 +1,16 @@
 // This assembly is included in the following assemblies:
 //
 // assembly-tracing.adoc
-[id='assembly-setting-up-tracing-mirror-maker-connect-{context}']
-= Setting up tracing for Mirror Maker and Kafka Connect
+[id='assembly-setting-up-tracing-mirror-maker-connect-bridge-{context}']
+= Setting up tracing for Mirror Maker, Kafka Connect and Kafka Bridge
 
-Distributed tracing is supported for Mirror Maker and Kafka Connect (including Kafka Connect with Source2Image support). 
+Distributed tracing is supported for Mirror Maker, Kafka Connect (including Kafka Connect with Source2Image support) and Kafka Bridge.
```
[the remark this answers is not in the document]
Comment (ppatierno) [reply documentation/assemblies/assembly-setting-up-tracing-mirror-maker-connect-bridge.adoc]: @laidan6000 why should we put `{ProductName}` here?

File: documentation/modules/proc-enabling-tracing-in-connect-mirror-maker-bridge-resources.adoc (hunk 5 above)
Comment (ppatierno) [reply documentation/modules/proc-enabling-tracing-in-connect-mirror-maker-bridge-resources.adoc]: @laidan6000 still here, why adding `{ProductName}` ?

File: documentation/modules/proc-enabling-tracing-in-connect-mirror-maker-bridge-resources.adoc (hunk 8 above)
Comment (ppatierno) [reply documentation/modules/proc-enabling-tracing-in-connect-mirror-maker-bridge-resources.adoc]: right

File: documentation/assemblies/assembly-setting-up-tracing-mirror-maker-connect-bridge.adoc [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+
+.Tracing in Mirror Maker
+
+For Mirror Maker, messages are traced from the source cluster to the target cluster; the trace data records messages entering and leaving the Mirror Maker component.
+
+.Tracing in Kafka Connect
+
+Only messages produced and consumed by Kafka Connect itself are traced. To trace messages sent between Kafka Connect and external systems, you must configure tracing in the connectors for those systems. For more information, see xref:assembly-deployment-configuration-kafka-connect-{context}[].
+
+.Tracing in the the Kafka Bridge
```
Comment (laidan6000) [inline documentation/assemblies/assembly-setting-up-tracing-mirror-maker-connect-bridge.adoc]: Repeated word: the

File: documentation/assemblies/assembly-setting-up-tracing-mirror-maker-connect-bridge.adoc [hunk 11]
```diff
@@ -0,0 +1,22 @@
...
+
+Messages produced and consumed by the Kafka Bridge are traced. Incoming HTTP requests from client applications to send and receive messages through the Kafka Bridge are also traced.
+In order to have and end-to-end tracing, you must configure tracing in your HTTP clients.
```
Comment (laidan6000) [inline documentation/assemblies/assembly-setting-up-tracing-mirror-maker-connect-bridge.adoc]:
```suggestion
In order to have end-to-end tracing, you must configure tracing in your HTTP clients.
```

**Pull request merged**